man/systemd.timer.xml

```diff
@@ -266,13 +266,13 @@
 
         <listitem><para>Delay the timer by a randomly selected, evenly distributed amount of time between 0
         and the specified time value. Defaults to 0, indicating that no randomized delay shall be applied.
-        Each timer unit will determine this delay randomly before each iteration, and the delay will simply
-        be added on top of the next determined elapsing time, unless modified with
-        <varname>FixedRandomDelay=</varname>, see below.</para>
+        Each timer unit will determine this delay randomly before each iteration, unless modified with
+        <varname>FixedRandomDelay=</varname>, see below. The delay is added on top of the next determined
+        elapsing time or the service manager's startup time, whichever is later.</para>
```

Comment:
that's not true? we do not take the startup time into account, do we? all we do is use the current time at the timer_enter_waiting() moment into account, which might resolve to something similar, but not necessarily? or did i miss somethign?